src/test/java/teammates/test/cases/action/FeedbackParticipantFeedbackResponseCommentDeleteActionTest.java

```diff
@@ -0,0 +1,263 @@
+package teammates.test.cases.action;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.FeedbackQuestionAttributes;
+import teammates.common.datatransfer.attributes.FeedbackResponseAttributes;
+import teammates.common.datatransfer.attributes.FeedbackResponseCommentAttributes;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.util.Const;
+import teammates.storage.api.FeedbackQuestionsDb;
+import teammates.storage.api.FeedbackResponseCommentsDb;
+import teammates.storage.api.FeedbackResponsesDb;
+import teammates.ui.controller.AjaxResult;
+import teammates.ui.controller.FeedbackParticipantFeedbackResponseCommentDeleteAction;
+import teammates.ui.pagedata.FeedbackResponseCommentAjaxPageData;
+
+/**
+ * SUT: {@link FeedbackParticipantFeedbackResponseCommentDeleteAction}.
+ */
+public class FeedbackParticipantFeedbackResponseCommentDeleteActionTest extends BaseActionTest {
+
+    @Override
+    protected String getActionUri() {
+        return Const.ActionURIs.FEEDBACK_PARTICIPANT_FEEDBACK_RESPONSE_COMMENT_DELETE;
+    }
+
+    @Override
+    protected void prepareTestData() {
+        super.prepareTestData();
+        dataBundle = loadDataBundle("/FeedbackParticipantFeedbackResponseCommentDeleteTest.json");
+        removeAndRestoreDataBundle(dataBundle);
+    }
+
+    @Override
+    @Test
+    public void testExecuteAndPostProcess() {
```

Comment:
This test is too long. You can separate the tests into different test cases functions and call them in this particular function. In that case will be more singular